cvat/apps/engine/migrations/0024_auto_20191023_1025.py

```diff
@@ -68,7 +68,10 @@ def migrate_task_data(db_task_id, db_data_id, original_video, original_images, s
                     original_chunk_writer = Mpeg4ChunkWriter(100)
                     compressed_chunk_writer = ZipCompressedChunkWriter(image_quality)
 
-                    for chunk_idx, chunk_images in enumerate(reader.slice_by_size(chunk_size)):
+                    counter = itertools.count()
```

Comment:
Will we have an issue with the code in the future? I don't see an alternative at the moment but if we change source code it may affect the migration.

Reply:
Potentially yes, but I cannot suggest an alternative at the moment except for duplicating the functions directly in the migration file.